cms/models.py

```diff
@@ -1069,6 +1073,31 @@ class Meta:
         FieldPanel("content"),
     ]
 
+    api_fields = [
+        APIField("child_pages", serializer=ProductChildPageSerializer()),
+        APIField("language"),
```

Comment:
Language is not getting serialized with name. Will need a custom serializer OR Property?